Makefile

```diff
@@ -1,19 +1,19 @@
 TEMP_TEST_OUTPUT=/tmp/contract-test-service.log
 
 build-contract-tests:
-	cd ./ContractTests && swift build
+	cd ./ContractTests && swift build --product ContractTests
 
 start-contract-test-service: build-contract-tests
-	cd ./ContractTests && swift run
+	cd ./ContractTests && swift run ContractTests serve -b 0.0.0.0:8080
 
 start-contract-test-service-bg:
 	@echo "Test service output will be captured in $(TEMP_TEST_OUTPUT)"
 	@make start-contract-test-service >$(TEMP_TEST_OUTPUT) 2>&1 &
 
 run-contract-tests:
 	@curl -s https://raw.githubusercontent.com/launchdarkly/sdk-test-harness/master/downloader/run.sh \
-      | VERSION=v2 PARAMS="-url http://localhost:8080 -debug -stop-service-at-end -skip-from ./ContractTests/testharness-suppressions.txt $(TEST_HARNESS_PARAMS)" sh
+      | VERSION=v2 PARAMS="-url http://0.0.0.0:8080 -debug -stop-service-at-end -skip-from ./ContractTests/testharness-suppressions.txt $(TEST_HARNESS_PARAMS)" sh
```

Comment:
Was localhost not working?

Reply:
It might actually work. I was running into some other problems originally with this and tried that first. I'll revert and see if it runs through.